charts/tigera-operator/README.md

```diff
@@ -89,6 +89,9 @@ The default values.yaml should be suitable for most basic deployments.
 ```
 # Image pull secrets to provision for pulling images from private registries.
 # This field is a map of desired Secret name to .dockerconfigjson formatted data to use for the secret.
+# Populates the `imagePullSecrets` property for all Pods controlled by the `Installation` resource.
+# This will also populate a Secret in the `tigera-operator` namespace, but Secrets in other namespaces managed
+#  by the operator (e.g. `calico-system`) will require creation through other means.
```

Comment:
> but Secrets in other namespaces managed
>  by the operator (e.g. `calico-system`) will require creation through other means.

This doesn't match my understanding. Namely, I expect the tigera/operator will propagate the image pull secrets specified here to all the Calico namespaces that need them.

Reply:
I removed the statement about Secrets